.github/workflows/drivers.yml

```diff
@@ -21,6 +21,12 @@ jobs:
     steps:
       - name: Checkout repository
         uses: actions/checkout@v4
+      - name: Fetch base branch
+        run: |
+          # Fetch the base branch to ensure its history is available for diffing
+          git fetch origin ${{ github.event.pull_request.base.ref }}:${{ github.event.pull_request.base.ref }}
+          # Only run this if it's a pull request and the base ref exists
+        if: github.event_name == 'pull_request' && github.event.pull_request.base.ref != ''
```

Comment:
to me, it reads better with `if:` comes before `run:`, your call if you want to change it